transmart-copy/src/main/groovy/org/transmartproject/copy/table/TreeNodes.groovy

```diff
@@ -137,7 +138,7 @@ class TreeNodes {
             log.info "Reading tree nodes from file ..."
             def insertCount = 0
             def existingCount = 0
-            LinkedHashMap<String, Class> header = columns
+            LinkedHashMap<String, Class> header
```

Comment:
Why do you have the variable declaration without initialization? There are several similar declarations above.

Reply:
Because of the changed type. This initialisation that was there was actually not used.